code/game/gamemodes/events/hivemind_invasion.dm

```diff
@@ -26,8 +26,9 @@
 /datum/event/hivemind/start()
 	var/turf/start_location
 	var/active_players = 0
-	for(var/mob/M in GLOB.player_list)
-		if(M.mind.assigned_role in list(JOBS_ANTI_HIVEMIND))
+	var/mob/living/carbon/human/figher
+	for(figher in GLOB.player_list)
```

Comment:
change figher to fighter? maybe?

Reply:
Yes, it should be fighter :p for the sake of spelling